kolibri/utils/server.py

```diff
@@ -52,7 +52,9 @@
 
 # This is a special file with daemon activity. It logs ALL stderr output, some
 # might not have made it to the log file!
-DAEMON_LOG = os.path.join(conf.KOLIBRI_HOME, "server.log")
+DAEMON_LOG = os.path.join(conf.LOG_ROOT, "daemon.txt")
+
+KOLIBRI_LOG = settings.LOGGING["handlers"]["file"]["filename"]
```

Comment:
This looks wrong - we are accessing Django settings in the module root here. The server module is imported in the module scope of the cli module, which is executed before Django is initialized. I am a little surprised this has not caused breakage.

Given that this value is only referenced in the `cli.py` file, I feel like we could do without this extra indirection, and just reference the value directly in `cli.py`.

Reply:
I wonder if it's because we have configured `DJANGO_SETTINGS_MODULE` to `kolibri.deployment.default.settings.base` in `conf.py`, we are just getting the variable LOGGING from the settings.base module?
But yeah I will reference this in cli.py directly

Reply:
When you do `from django.conf import settings` - the `settings` object is not actually the direct settings, but a Django `LazySettings` object. As soon as you try to access a key from the settings object, it will check that it is initialized, if it is not, it will throw an error. That's why I was confused!

What will probably be good in the future is actually to move our logging configuration out of the settings `base.py` and into a separate file so that we can reference it independently of the settings (and also avoid the repetition in `options.py` too). This will have the slight issue of causing problems if someone completely overrides our base settings, but that's their problem, not ours!

Reply:
ah it makes sense to me now. thank you so much for the explanation!